service/create/service.go

```diff
@@ -492,10 +519,21 @@ func (s *Service) runMachine(input runMachineInput) error {
 			},
 		},
 	}); err != nil {
-		return microerror.MaskAny(err)
+		return nil, microerror.MaskAny(err)
 	}
 
 	s.logger.Log("info", fmt.Sprintf("instance '%s' tagged", input.name))
+	instance := reservation.Instances[0]
 
-	return nil
+	return instance, nil
```

Comment:
Ok cool. Then it is up to you. I raised my concerns and elaborated on my thoughts. Go ahead on your own behalf. So we have these options. 

- Keep it as it is. And create an issue to track the design flaw in the boards. 
- Provide separate methods to get the design more right now.